mempool/cat/store.go

```diff
@@ -142,21 +142,22 @@ func (s *store) getTxsBelowPriority(priority int64) ([]*wrappedTx, int64) {
 
 // purgeExpiredTxs removes all transactions that are older than the given height
 // and time. Returns the amount of transactions that were removed
```

Comment:
```suggestion
// and time. Returns the purged txs and amount of transactions that were purged.
```

still seems like purge could be renamed to evicted